### ISSUE TITLE
Default date range shown is year 1 A.D. to present

### ISSUE THREAD

**Issue by rlue** [beancount/fava#1194]
I'm not sure exactly when this started happening, but whenever I load my beancount journal in fava, the default view of the income statement / balance sheet charts covers the date range 1 – 2020 A.D. (as in, just over two thousand years). As a result, the charts appear empty because I have no transactions recorded for the first 99.8% of that range:

![Screenshot_20201222_213246](https://user-images.githubusercontent.com/12194123/102894573-477fa000-449e-11eb-97af-fba356021433.png)

A potential cause is hinted at in the screenshot above, where focusing on the "Time" input field auto-suggests years for which I have journal entries recorded: 2020, 2019, 2018, 2017, 1986, and... 1? But if I select "1", I get an error:

![Screenshot_20201222_214234](https://user-images.githubusercontent.com/12194123/102894828-a6ddb000-449e-11eb-9f61-3d5278eb58f7.png)

> Failed to parse date: 1

Moreover, if I go to the Journal page, I can see that there are no entries (transactions, account openings/closings, notes, etc.) that predate 1986:

![Screenshot_20201222_213437](https://user-images.githubusercontent.com/12194123/102894920-c7a60580-449e-11eb-8f37-a7b7e64d6507.png)

I've been using fava for years now and have never had this problem. I check it multiple times a week, and first noticed this problem last Friday. I've tried reverting my journal to the state it was in one month ago, but the problem persists, so I suspect it's related to fava and not to recent modifications to my journal.

Sorry in advance if it's something obvious—I'm not what you would call an expert user of fava, but I'm pretty well stumped.

**Comment by yagebu** [comment]
Can you provide an example Beancount file to reproduce this? Which versions of Fava and Beancount do you have installed?

The logic to generate these available years is rather simple, so the most straightforward explanation would be some entry in the year 1.

> Moreover, if I go to the Journal page, I can see that there are no entries (transactions, account openings/closings, notes, etc.) that predate 1986

Are all entry types active in that screenshot? open/close are not shown by default for example

**Comment by rlue** [comment]
> Are all entry types active in that screenshot? open/close are not shown by default for example

Well that is terribly embarrassing. I had some price directives (at the top of the document, no less!) that were set to the year 1.

Very sorry to bother!

**Comment by addisonklinke** [comment]
@rlue what about cases where you have a small handful of historical transactions that are correctly dated, but still way in the past? For example, some equities or real estate I purchased 5 years before I started using beancount. I'd like the transaction dates to be accurate for tax purposes, but that fills the majority of my income statement and balance sheet reports with blank space. Is there a setting to tell Fava to only display from a given date onwards?
